mails/themes/classic/core/bankwire.html.twig

```diff
@@ -0,0 +1,84 @@
+{% extends '@MailThemes/classic/components/layout.html.twig' %}
+
+{% block content %}
+<tr>
+  <td align="center" class="titleblock">
+    <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
+      <span class="title">{{ 'Hi {firstname} {lastname},'|trans({}, 'EmailsBody', locale)|raw }}</span><br/>
```

Comment:
Why raw for firstname and shop name?

Reply:
I updated the templates so that `raw` is not used when there are variables (unless it contains html code).